files/zh-cn/learn/tools_and_testing/cross_browser_testing/your_own_automation_environment/index.md

```diff
@@ -257,26 +241,41 @@ driver.get("http://localhost:8888/fake-div-buttons.html");
 
 使用一个远程的服务器地址代码会更灵活——当你启用远程服务器运行测试时，如果企图使用本地路径，代码会中断。
 
-现在添加下面代码到 `quick_test.js` 的最后：
+更新 `example()` 函数如下：
 
 ```js
-driver.get(
-  "https://mdn.github.io/learning-area/tools-testing/cross-browser-testing/accessibility/native-keyboard-accessibility.html",
-);
+const { Builder, Browser } = require("selenium-webdriver");
+
+(async function example() {
+  const driver = await new Builder().forBrowser(Browser.FIREFOX).build();
+  driver.get(
+    "https://mdn.github.io/learning-area/tools-testing/cross-browser-testing/accessibility/native-keyboard-accessibility.html",
+  );
+})();
 ```
 
 ### 同文档交互
 
-得到测试的文档后，我们就需要同它进行交互操作了，比如获取特定的元素对它进行某种测试。WebDriver 提供了[通过多种方法选择 UI 元素](https://www.selenium.dev/zh-cn/documentation/webdriver/elements/)，比如通过 ID、class、元素名称等。具体的选择通过 `findElement()` 方法完成，只需要将选择的元素作为参数传给它即可。举例来说，通过 ID 选择一个元素：
+得到测试的文档后，我们就需要同它进行交互操作了，比如获取特定的元素对它进行某种测试。WebDriver 提供了[多种方法来选择 UI 元素](https://www.selenium.dev/zh-cn/documentation/webdriver/elements/)，比如通过 ID、class、元素名称等。具体的选择通过 `findElement()` 方法完成，只需要将选择的元素作为参数传给它即可。举例来说，通过 ID 选择元素：
 
 ```js
 const element = driver.findElement(By.id("myElementId"));
 ```
 
-通过 CSS 查找一个元素的最常用方法是使用 By.css 方法，它可以通过 CSS 选择器的方式来选择元素。把下面代码敲到 `quick_test.js` 的最后面：
+通过 CSS 查找元素的最常用方法是使用 `By.css()` 方法，它可以通过 CSS 选择器的方式来选择元素。
+
+现在更新 `example()` 函数如下：
 
 ```js
-const button = driver.findElement(By.css("button:nth-of-type(1)"));
+const { Builder, Browser, By } = require("selenium-webdriver");
+
+(async function example() {
+  const driver = await new Builder().forBrowser(Browser.FIREFOX).build();
+  driver.get(
+    "https://mdn.github.io/learning-area/tools-testing/cross-browser-testing/accessibility/native-keyboard-accessibility.html",
+  );
+  const button = driver.findElement(By.css("button:nth-of-type(1)"));
+})();
 ```
 
 ### 测试元素
```

Comment:
L283 should use localized link

https://www.selenium.dev/zh-cn/documentation/webdriver/elements/information/#%E6%96%87%E6%9C%AC%E5%86%85%E5%AE%B9